Sublime Text/Sublime Text 4.download.recipe

```diff
@@ -73,22 +69,31 @@ limitations under the License.</string>
 		<dict>
 			<key>Arguments</key>
 			<dict>
-				<key>input_plist_path</key>
-				<string>%RECIPE_CACHE_DIR%/%NAME%/%NAME%.app/Contents/Info.plist</string>
+				<key>input_path</key>
+				<string>%RECIPE_CACHE_DIR%/%NAME%/Sublime Text.app</string>
+				<key>requirement</key>
+				<string>identifier "com.sublimetext.4" and anchor apple generic and certificate 1[field.1.2.840.113635.100.6.2.6] /* exists */ and certificate leaf[field.1.2.840.113635.100.6.1.13] /* exists */ and certificate leaf[subject.OU] = Z6D26JE4Y4</string>
+				<key>strict_verification</key>
+				<true/>
 			</dict>
 			<key>Processor</key>
-			<string>Versioner</string>
+			<string>CodeSignatureVerifier</string>
 		</dict>
 		<dict>
 			<key>Arguments</key>
 			<dict>
-				<key>input_path</key>
-				<string>%RECIPE_CACHE_DIR%/%NAME%/%NAME%.app</string>
-				<key>requirement</key>
-				<string>identifier "com.sublimetext.4" and anchor apple generic and certificate 1[field.1.2.840.113635.100.6.2.6] /* exists */ and certificate leaf[field.1.2.840.113635.100.6.1.13] /* exists */ and certificate leaf[subject.OU] = Z6D26JE4Y4</string>
+				<key>info_path</key>
+				<string>%RECIPE_CACHE_DIR%/%NAME%/Sublime Text.app/Contents/Info.plist</string>
+				<key>plist_keys</key>
+				<dict>
+					<key>CFBundleVersion</key>
```

Comment:
`Versioner` was pulling `CFBundleShortVersionString`, can we switch `PlistReader` to do the same for consistency?

Reply:
I'm not sure I understand why you'd want that. Short version string has the word 'Build' in it whereas the cfbundleversion is just the number. Much easier to compare to for whether or not a version is newer than what's installed

Reply:
Thinking of downstream jss recipe behavior since Jamf Pro uses `CFBundleShortVersionString` in inventory collection, but I suppose I could account for that in the recipe override itself. Agreed that the numerical string is easier to parse, happy to stick with that. Just need to update the overrides to account for the download recipe changes, but I can do so in a later commit. Thanks for the PR!